src/main/java/pl/project13/maven/git/GitCommitIdMojo.java

```diff
@@ -66,7 +66,11 @@
   // these properties will be exposed to maven
   public static final String BRANCH = "branch";
   public static final String DIRTY = "dirty";
-  public static final String COMMIT_ID = "commit.id.full";
+  @Deprecated
+  public static final String COMMIT_ID_OLD = "commit.id";
```

Comment:
the intention of the PR is to keep the property around. However I think we should drop it somewhere in the feature and therefore I would say it's deprecated